src/tcgwars/logic/impl/gen1/WizardsBlackStarPromosNG.groovy

```diff
@@ -152,8 +152,7 @@ public enum WizardsBlackStarPromosNG implements LogicCardInfo {
           energyCost C
           attackRequirement {}
           onAttack {
-            reduceDamageNextTurn(hp(10), thisMove)
-            // I don't understand the benching condition
```

Comment:
The Benching condition here is different than other effects;

- If there is `benching either pokemon ends this effect` is present, then the delayed effect is cleared for either switch. 
- If not (default case) then the effect only ends when self was switched. 

I think this should also be added to the method definition. 

`if(either)` -> add `after SWITCH, defending, {unregister()}` as well